jekyll/_docs/build-image-trusty.md

```diff
@@ -6,8 +6,9 @@ description: Ubuntu 14.04 (Trusty)
 changefreq: "weekly"
 ---
 
-You can run your Linux builds on Ubuntu 14.04 Trusty (the default). You can 
-switch to Trusty from "Project Settings" -> "Build Environment" of your project.
+You can run your Linux builds on the default image, Ubuntu 14.04 (Trusty). You 
+can switch to Trusty from "Project Settings" -> "Build Environment" of your 
+project.
```

Comment:
Ok, if that's the case, then I think the best way to write this is:

"The default image is Ubuntu 14.04 (Trusty), but you can always change the image directly by going to 'Project Settings' -> 'Build Environment'."

This captures the (new) default and also shows people where to change that image if they need to.